### DISCUSSION TITLE
any java script to use in rigtools

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#1251] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
are there any code to run in the JavaScript inject part were you can inject java script  is there any good things to use

**Comment by akabutnicer** [comment, marked as answer]
you have access to the developer tools interface; however, I'm not exactly sure what it can do. ill check later today and give you an update

**Reply by buddyted** [reply]
like this here
![Screenshot 2024-11-08 11 03 35 AM](https://github.com/user-attachments/assets/99ee818d-a6e0-4148-8ae3-ab97c0d36ec1)

**Reply by buddyted** [reply]
to double check

**Reply by akabutnicer** [reply]
what version of rigtools is this? anyways, there is a working way you can modify an extension's manifest remotely. though I can't expose it as it's not fully complete yet.

**Reply by buddyted** [reply]
okay and I'm on v125 on my Chromebook if that says anything how do I check that

**Reply by buddyted** [reply]
![Screenshot 2024-11-08 11 08 20 AM](https://github.com/user-attachments/assets/96a860f9-587d-4302-9d91-3cd4090dbd65)

**Reply by akabutnicer** [reply]
i meant this rigtools interference looked different to me, I was wondering who made it

**Reply by buddyted** [reply]
does it say it on the corner of the interferecne

**Reply by buddyted** [reply]
silly goober money gang

**Reply by buddyted** [reply]
> i meant this rigtools interference looked different to me, I was wondering who made it

So what do you mean

**Reply by akabutnicer** [reply]
> > i meant this rigtools interference looked different to me, I was wondering who made it
> 
> So what do you mean

Yeah I never heard of them before.  Anyways, whats the websocket URL present?

**Reply by buddyted** [reply]
> > > i meant this rigtools interference looked different to me, I was wondering who made it
> > 
> > 
> > So what do you mean
> 
> Yeah I never heard of them before. Anyways, whats the websocket URL present?

![Screenshot 2024-11-08 5 13 03 PM](https://github.com/user-attachments/assets/785ddc15-8a38-44ba-a73a-b8e34ec45cfd)
this?

**Reply by Sam-hacks1** [reply]
it looks like that for me too

**Comment by schoolexploitkid** [comment]
/// execute_script.js
window.addEventListener("keyup", event => {
  if (event.ctrlKey && event.which === 192) {
    let code = prompt("Eval:");
    if (code.startsWith("javascript:")) {
      code = code.substring(11);
    }
    eval(code);
  }
});

**Reply by akabutnicer** [reply]
he means on the targeted extension, genius

**Reply by buddyted** [reply]
What does that do?

**Reply by akabutnicer** [reply]
nothing that could help you

**Comment by ghost** [comment]
you could do tr3nch if you have one of the extensions that has that type of permission.

**Reply by buddyted** [reply]
what extension will I need for that

**Reply by akabutnicer** [reply]
if you have flags enabled, I have came up with a way to implement tr3nch in rigtools

**Reply by ghost** [reply]
> if you have flags enabled, I have came up with a way to implement tr3nch in rigtools

how so? wait this: Extensions on chrome:// URLs?
